Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
```

Comment:
Can we use `focal` here? As we will use Ubuntu 20.04 in our deployment for actual IOI

(or `ubuntu:20.04` for more clarity)

Reply:
Done.